pwa/src/components/common/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import "./breadcrumbs.css";
+import { Link } from "gatsby";
+
+interface BreadcrumbsProps {
+  crumbs: {
+    pathname: string;
+    crumbLabel: string;
+  }[];
+}
+
+const Breadcrumbs: React.FC<BreadcrumbsProps> = ({ crumbs }) => {
+  const [mappedCrumbs, setMappedCrumbs] = React.useState(null);
+  const crumbsLength = crumbs.length - 1;
+
+  React.useEffect(() => {
+    const splitPathname = location.pathname.split("/");
+
+    const _mappedCrumbs = crumbs.map((crumb, idx) => {
+      const crumbLabel = crumb.crumbLabel;
+      let pathname = "/";
+      for (let i = 0; i <= idx; i++) {
+        pathname += splitPathname[i] ? `${splitPathname[i]}/` : splitPathname[i];
+      }
+      return { ...{ pathname, crumbLabel } };
+    });
+
+    setMappedCrumbs(_mappedCrumbs);
+  }, [crumbs]);
+
+  return (
+    <ul className="breadcrumbs">
+      {mappedCrumbs?.map((crumb, idx) => {
+        return (
+          <li key={idx}>
+            <Link to={crumb.pathname}>{crumb.crumbLabel.charAt(0).toUpperCase() + crumb.crumbLabel.slice(1)}</Link>
```

Comment:
Because we have the `lodash` package available in the application, we can simplify this line.

```
import * as _ from "lodash";
...
_.upperFirst(crumb.crumbLabel)
```

Instead of having to `.toUpperCase()` and `slice()` everything manually 👍🏼